### ISSUE TITLE
show PSNR  values

### ISSUE THREAD

**Issue by Bueeie** [tegg89/SRCNN-Tensorflow#39]
Hi, may I know how to show the super resolution PSNR values?

Thanks! :)

**Comment by FJstyle** [comment]
Do you know it, now?

**Comment by Bueeie** [comment]
Yes. You need to resize the result image to the size of the ground truth. Then only can get the PSNR value.

**Comment by dingchengshan** [comment]
Hi,Can you give an example?

Thank you!
